pkg/otlp/metrics/metrics_translator.go

```diff
@@ -43,7 +43,10 @@ const (
 )
 
 var (
-	signalTypeSet = attribute.NewSet(attribute.String("signal", "metrics"))
+	signalTypeSet      = attribute.NewSet(attribute.String("signal", "metrics"))
+	rateAsGaugeMetrics = map[string]struct{}{
+		"placeholder.rate.as.gauge.metric": {},
```

Comment:
You could override this map in tests and avoid having this in actual code (if different tests expected different values on the map, then moving this inside the translator may make sense, but I think it's not necessary right now AFAICT)

Reply:
Done, I've overriden `rateAsGaugeMetrics` in tests and left it empty in `metrics_translator.go`. Once I've filled `rateAsGaugeMetrics` in a subsequent PR, I think I will be able to remove the overrides in test and just update [here](https://github.com/DataDog/opentelemetry-mapping-go/blob/aabd94ecc082197c4c7ff94c5f07cb335feb111f/pkg/otlp/metrics/metrics_translator_test.go#L271) with one of the "real" metrics, as this is unlikely to change.

Reply:
Makes sense, thanks!